doc/toc.yml

```diff
@@ -7,6 +7,6 @@ items:
 - name: Controls
   items:
     - name: CardContentControl
```

Comment:
maybe change both title and file name to "Cards"?
```suggestion
    - name: Cards
```

Reply:
good catch thanks !